### DISCUSSION TITLE
ScriptReference/Playables.PlayableExtensions.SetTime

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#616] [Notes]
# ScriptReference/Playables.PlayableExtensions.SetTime

https://docs.unity3d.com/ScriptReference/Playables.PlayableExtensions.SetTime.html

**Comment by YondernautsGames** [comment]
If using this with an animation playable then it will break root motion. Root motion seems to be based on the difference in time between the last 2 calls, so to get around this you can call the method twice yourself.

For example, if you want to set the time to a specific value you use:

```
myPlayable.SetTime(targetTime - Time.deltaTime);
myPlayable.SetTime(targetTime);
```
